background.ts

```diff
@@ -481,6 +482,44 @@ Electron.ipcMain.handle('service-forward', async(event, service, state) => {
   }
 });
 
+Electron.ipcMain.on('k8s-integrations', async(event) => {
+  if (k8smanager instanceof WSLBackend) {
+    event.reply('k8s-integrations', await k8smanager?.listIntegrations());
+  }
+});
+
+Electron.ipcMain.on('k8s-integration-set', async(event, name, newState) => {
+  if (k8smanager instanceof WSLBackend) {
```

Comment:
Check [`k8smanager.backend`](https://github.com/rancher-sandbox/rancher-desktop/blob/732e876b3dc66c571e62688455eead18f8a61fc4/src/k8s-engine/k8s.ts#L108) instead; that's just a string (and means we don't need to import `WSLBackend` at all, I think).

Reply:
![image](https://user-images.githubusercontent.com/19395776/163471037-292b080c-d29c-4840-9cb1-1ff8565feef8.png)

Typescript complains :cry:

Reply:
Ah, sorry, I didn't realize `listIntegrations` only exists on the WSL backend now. In that case, yes, we need to actually import.

Sorry about that.